Description: Removed a failing manpage macro.
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2016-11-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/asn1c/asn1c.1
+++ b/asn1c/asn1c.1
@@ -1,4 +1,4 @@
-.TH ASN1C 1 "\*(Dt" "ASN.1 Compiler" "ASN.1 Compiler"
+.TH ASN1C 1 "" "ASN.1 Compiler" "ASN.1 Compiler"
 .SH NAME
 asn1c \- ASN.1 Compiler
 .SH SYNOPSIS
--- a/asn1c/enber.1
+++ b/asn1c/enber.1
@@ -8,7 +8,7 @@
 .fi
 .sp
 ..
-.TH ENBER 1 "\*(Dt" "unber counterpart" "unber counterpart"
+.TH ENBER 1 "" "unber counterpart" "unber counterpart"
 .SH NAME
 enber \- Convert the unber XML output back into BER
 .SH SYNOPSIS
--- a/asn1c/unber.1
+++ b/asn1c/unber.1
@@ -8,7 +8,7 @@
 .fi
 .sp
 ..
-.TH UNBER 1 "\*(Dt" "ASN.1 BER Decoder" "ASN.1 BER Decoder"
+.TH UNBER 1 "" "ASN.1 BER Decoder" "ASN.1 BER Decoder"
 .SH NAME
 unber \- ASN.1 BER Decoder
 .SH SYNOPSIS
